decidim-proposals/app/helpers/decidim/proposals/application_helper.rb

```diff
@@ -172,7 +172,7 @@ def activity_filter_values
       def filter_origin_values
         origin_values = []
         origin_values << TreePoint.new("official", t("decidim.proposals.application_helper.filter_origin_values.official")) if component_settings.official_proposals_enabled
-        origin_values << TreePoint.new("citizens", t("decidim.proposals.application_helper.filter_origin_values.citizens"))
+        origin_values << TreePoint.new("participants", t("decidim.proposals.application_helper.filter_origin_values.participants"))
```

Comment:
Maybe you need to change or remove the key for other languages as well?

Reply:
Crowdin will take care of it automatically after this is merged in `develop`